Add Launch on qBraid Button

rryoung98 opened the pull request:
With Launch on qBraid, users will be able to clone the benchmarking repository to their qBraid Lab with a click of a button, taking them from GitHub to qBraid Lab. This will help the users to run the notebooks in as little as five clicks.

rtvuser1 commented:
This is an interesting addition and we are considering where best to place this reference.  qBraid is a little different from the other open-sources packages referenced in the top-level docs, in that it is a paid product that requires a user subscription. We support referencing such products, as many are very useful, but we need to do it in a consistent fashion across all, probably not in the top-level README.  For example, with qBraid, you would probably want to have more than a single sentence. You'd probably want to include a short paragraph that informs users of the steps to take or the options to select.

rryoung98 commented:
Hello @rtvuser1  thank you for the thoughtful comment. 

qBraid's free tier would not require a purchase of subscription and the ability to open notebooks to run freely on our servers. We hope that this can greatly increase the amount of individuals trying out the benchmarking notebooks.

I've updated and added it to the `_setup` folder as a singular README.md about how to use qBraid.

rtvuser1 commented:
This is better.  The PR will be merged followed by a few improvements I will make to the README itself to make it more consistent with others.
